tests/pathfinding/test_capacity_updates.py

```diff
@@ -0,0 +1,409 @@
+"""
+The tests in this module mock Capacity Updates and call on_pfs_update().
+
+The Capacity Updates show different correct and incorrect values to test all edge cases
+"""
+
+from typing import List
+
+import pytest
+from eth_utils import decode_hex
+
+from pathfinding_service import PathfindingService
+from pathfinding_service.exceptions import InvalidCapacityUpdate
+from pathfinding_service.model import TokenNetwork
+from raiden.constants import UINT256_MAX
+from raiden.messages import UpdatePFS
+from raiden.utils import CanonicalIdentifier
+from raiden.utils.typing import (
+    ChainID,
+    ChannelID,
+    Nonce,
+    Signature,
+    TokenAmount,
+    TokenNetworkAddress,
+)
+from raiden_libs.types import Address, ChannelIdentifier
+
+
+DEFAULT_TOKEN_NETWORK_ADDRESS = Address('0x6e46B62a245D9EE7758B8DdCCDD1B85fF56B9Bc9')
+DEFAULT_TOKEN_NETWORK_ADDRESS_BYTES = TokenNetworkAddress(
+    decode_hex(DEFAULT_TOKEN_NETWORK_ADDRESS),
+)
+DEFAULT_TOKEN_ADDRESS = Address('0x44Ac22fd9672cC559Ab171603D474cEA8a2D7b4D')
+
+
+def get_UpdatePFS_message(
```

Comment:
The naming here looks wrong. I get it's to get the class but if you go for snake case, please stick to snake case. Mixing Camelcase and snake_case in the same object name looks wrong.

Reply:
I am a bit a newbee, but wtf is a camel and a snake case?

Reply:
@Dominik1999 

**Snake case**: `def a_function_that_adds_two_numbers()`
**Camel case**: `def AFunctionThatAddsTwoNumbers()`

They are two different naming style methods. We don't mix those.

https://blog.lmorchard.com/2013/01/23/naming-conventions/

Reply:
a now I get it, changed the function name